HackingWithSwift/SwiftUI/myProject2/myProject2/myProject2/ContentView.swift

```diff
@@ -0,0 +1,201 @@
+                            .background(.black)
+                            .clipShape(.buttonBorder)
+                            .font(.system(size: 70))
+                            
+                            Button("✌️"){
+                                counter += 1
+                                scissons()
+                            }
+                            .padding(20)
+                            .background(.black)
+                            .clipShape(.buttonBorder)
+                            .font(.system(size: 70))
+                        }
+                    }
+                    
+                    VStack{
+                        Spacer()
+                        Text("Score: \(score)")
+                    }
+                    .font(.system(size: 40))
+                }
+            }
+            .alert("Game over", isPresented: $endGame) {
+                Button ("Restart") {}
+            } message: {
+                Text("\(finalScore)")
+            }
+            .navigationTitle("Rock, Paper, Scissons!")
+        }
+    }
+    
+    func compChoose() -> some View {
+        Text(optionToСhoice[computerСhoice])
+            .font(.largeTitle.weight(.semibold))
+    }
+    
+    func rock()  {
+        if computerСhoice == 0 {
+            myChoise = "✊"
+            score += 1
+            newGame()
+            rightСhoice = true
+        }
+        else if computerСhoice == 1 {
+            myChoise = "✊"
+            score -= 1
+            newGame()
+        } else {
+            myChoise = "✊"
+            score += 1
+            newGame()
+            rightСhoice = true
+        }
+    }
+    
+    func paper() {
+        if computerСhoice == 0 {
+            myChoise = "✋"
+            score += 1
+            newGame()
+            rightСhoice = true
+        }
+        else if computerСhoice == 1 {
+            myChoise = "✋"
+            score += 1
+            newGame()
+            rightСhoice = true
+        } else {
+            myChoise = "✋"
+            score -= 1
+            newGame()
+        }
+    }
+    
+    func scissons() {
+        if computerСhoice == 0 {
+            myChoise = "✌️"
+            score -= 1
+            newGame()
+        }
+        else if computerСhoice == 1 {
+            myChoise = "✌️"
+            score += 1
+            newGame()
+            rightСhoice = true
+        } else {
+            myChoise = "✌️"
+            score += 1
+            newGame()
+            rightСhoice = true
+        }
+    }
+    
+    func myturn() -> some View {
+        Text("\(myChoise)")
+            .font(.system(size: 150))
+            .padding(25)
+            .background(Color.secondary)
+            .clipShape(.circle)
+            .shadow(radius: 40)
+    }
+    
+    func goodChoise() -> some View {
+        if rightСhoice == true {
+            Text("You win")
+        } else {
+            Text("You lose")
+        }
+    }
+    
```

Comment:
It is better to have functions that return a view right after `var body: some View`